test/unit/general/contextBrokerKeystoneSecurityAccess-test.js

```diff
@@ -38,6 +38,7 @@ var iotAgentLib = require('../../../lib/fiware-iotagent-lib'),
             port: 4041
         },
         authentication: {
+            type: 'keystone',
```

Comment:
@fgalan to proof backward compatibility, i can remove this line ;)

Reply:
Good idea. But instead of completealy remove it, just remove it, i.e:

```
// Next line is syntactically correct from a configuration point of view, but we comment it in order to ensure backward compability
// type: keystone
```

or a similar wording.

Reply:
fix by 871e6ef